### ISSUE TITLE
Open to Public Consumption

### ISSUE THREAD

**Issue by jordantrizz** [robbmanes/seeding-reward-bot#35]
Hello, I'm just wondering if this repository is open for anyone to utilize or is it just for Glow?

**Comment by robbmanes** [comment]
:wave: hi @jordantrizz; it's open to public consumption as per the [GPLv3](https://github.com/robbmanes/seeding-reward-bot/blob/main/LICENSE.md).  I'm an open-source developer advocate and strongly encourage you to adhere to the license when using `seeding-reward-bot` and further any fixes/features/code contributions should please be added back to the bot so everyone and every community can benefit from the code here.

I have an agreement with Glow and my workplace (which is a FOSS developer) for providing this kind of software with these licenses, in addition to my own ethical views.

Feel free to use the bot in your community, and don't hesitate to open GH issues for problems or contribute code back to the main repo!

TL;DR:
- You're absolutely free to use this software in your own community adhering to the GPLv3 license.
- I am a member of [Glow's Battlegrounds](https://discord.gg/glows) first, so my personal feature development patterns reflect what that community's needs are.
- If you make improvements/changes to the code, the license requires you to submit them back to this project so everyone can benefit.

Hope this helps!

**Comment by jordantrizz** [comment]
Thanks Rob. I'm seeing that there's additional features on Glows discord which isn't preset here. That's why I asked.

**Comment by robbmanes** [comment]
> Thanks Rob. I'm seeing that there's additional features on Glows discord which isn't preset here. That's why I asked.

Glow runs a lot of bots I don't maintain, but they're all named under the bot name of "Glowbot"; so the actual `#seeding` channel is the only place this particular bot in this github repo runs in.  I've limited the scope of just this bot to just seeding rewards/maintenance for HLL, and I don't work on any of the other bots so I'm unaware of their full functionality.

Most of the other bots are FOSS, but many are not; the achievements and performance-based-VIP bots are examples of non-FOSS bots that I don't have any access to, they're entirely run by Glow himself.  I help sysadmin the systems but that's about it.  Hope that helps add context.

**Comment by robbmanes** [comment]
Closing this if there's not any other questions; yell at me if there is or reach out to me in the HLL/Glow's discord.  <3
